frontend/src/pages/MeetingRoom/MeetingRoom.spec.tsx

```diff
@@ -132,6 +135,7 @@ describe('MeetingRoom', () => {
       getAllMediaDevices: vi.fn(),
       allMediaDevices,
     });
+    (usePublisherOptions as Mock).mockReturnValue({});
```

Comment:
this needed to be mocked due to this: https://github.com/Vonage/vonage-video-react-app/pull/139/files#diff-98d48a076c77e93ec230d6f50b3851675c8168242c0031490ed6a918f4d87f5eR62-R65